src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -340,6 +387,25 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                     onChange={this.onAddressChange}
                     domesticOnly={artwork.onlyShipsDomestically}
                     shippingCountry={artwork.shippingCountry}
+                    showPhoneNumberInput={false}
```

Comment:
since this is `shipping` shouldn't this be `true` and remove following `PhoneNumberForm` and let `AddressForm` handle it? alternatively maybe we should just move `phoneNumber` out of `AddressForm` altogether and in places we show `PhoneNumber` we add it as a separate component?
My main question is, when `showPhoneNumber` will be `true` if we don't use it here either.

Reply:
@ashkan18 it's needed in the bid flow!

Reply:
I just created: https://artsyproduct.atlassian.net/browse/PURCHASE-1750

It's caused a lot of confusion that the `AddressForm` code exists in the `Order` app, since it's used in a few places. We should move it somewhere more generic!